inference-engine/thirdparty/clDNN/src/impls/ocl/experimental_detectron_roi_feature_extractor.cpp

```diff
@@ -0,0 +1,88 @@
+        }
+        args.output = instance.output_memory_ptr();
+
+        return args;
+    }
+
+    event::ptr execute_impl(const std::vector<event::ptr>& events,
+                            experimental_detectron_roi_feature_extractor_inst& instance) override {
+        instance.copy_rois_input_to_second_output();
+        return parent::execute_impl(events, instance);
+    }
+
+public:
+    static primitive_impl* create(const experimental_detectron_roi_feature_extractor_node& arg) {
+        const auto output_layout = arg.get_output_layout();
+        const auto padding_filling_value = output_layout.data_padding.filling_value();
+        CLDNN_ERROR_NOT_EQUAL(arg.id(),
+                              "experimental_detectron_roi_feature_extractor padding filling value",
+                              padding_filling_value,
+                              "padding mode",
+                              0.0f,
+                              "Unknown padding mode in experimental_detectron_roi_feature_extractor.");
+
+        auto params = get_default_params<kernel_selector::experimental_detectron_roi_feature_extractor_params>(arg);
+        auto optional_params = get_default_optional_params<kernel_selector::experimental_detectron_roi_feature_extractor_optional_params>(arg.get_program());
+
+        const auto& primitive = arg.get_primitive();
+        size_t number_of_inputs = primitive->input_size() - 1;
+        for (std::size_t i = 1; i < number_of_inputs; i++) {
+            params.inputs.push_back(convert_data_tensor(arg.input(i).get_output_layout()));
+        }
+
+        params.output_dim = primitive->output_dim;
+        params.pooled_height = primitive->pooled_height;
+        params.pooled_width = primitive->pooled_width;
+        params.pyramid_scales = primitive->pyramid_scales;
+        params.sampling_ratio = primitive->sampling_ratio;
+        params.aligned = primitive->aligned;
+        params.number_of_inputs = number_of_inputs;
+
+        auto& kernel_selector = kernel_selector::experimental_detectron_roi_feature_extractor_kernel_selector::Instance();
+        auto best_kernels = kernel_selector.GetBestKernels(params, optional_params);
+
+        CLDNN_ERROR_BOOL(arg.id(), "best_kernels.empty()", best_kernels.empty(), "Cannot find a proper kernel with this arguments");
+        return new experimental_detectron_roi_feature_extractor_impl(arg, best_kernels.front());
+    }
+};
+
+namespace detail {
+attach_experimental_detectron_roi_feature_extractor_impl::attach_experimental_detectron_roi_feature_extractor_impl() {
+    implementation_map<experimental_detectron_roi_feature_extractor>::add(impl_types::ocl,
+                                                                            experimental_detectron_roi_feature_extractor_impl::create, {
+                                                                                std::make_tuple(data_types::f16, format::bfyx),
```

Comment:
What about support of i8/u8?

Reply:
According to operation specification floating-point type is supported